src/app/examples/badge-example/badge-example.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="badge-examples">
+  <kirby-badge colortype="danger">1</kirby-badge>
+  <kirby-badge colortype="alert">99</kirby-badge>
+  <kirby-badge colortype="primary">2</kirby-badge>
+  <kirby-badge colortype="secondary">3</kirby-badge>
+  <kirby-badge colortype="alert" text="Some text"></kirby-badge>
```

Comment:
Consider adding example with icon as well:
```suggestion
  <kirby-badge colortype="alert">
    <kirby-icon name="close"></kirby-icon>
  </kirby-badge>
```